README.md

```diff
@@ -164,23 +179,23 @@ cd boiler-plate-api
 docker build .
 ```
 
-This will create the Boilerplate API image and pull in the necessary dependencies. Once done, run the Docker and map the port to whatever you wish on your host 4005 in this case:
+This will create the Boilerplate API image and pull in the necessary dependencies. Once done, run the Docker and map the port to whatever you wish on your host 40001 in this case:
```

Comment:
This will create the Boilerplate API image and pull in the necessary dependencies. Once done, run the Docker image and map the port to whatever you wish on your host 40001 in this case: